pcdsdevices/lamp_motion.py

```diff
@@ -73,6 +73,12 @@ class LAMPMagneticBottle(BaseInterface, GroupDevice):
     magnet_y = Cpt(BeckhoffAxis, ':MMS:06', kind='normal')
     magnet_z = Cpt(BeckhoffAxis, ':MMS:04', kind='normal')
 
+    # TIXEL Manipulator Motors
+    tixel_x = Cpt(BeckhoffAxis, ':MMS:14', kind='normal',
+                  doc='Tixel manipulator x-axis')
```

Comment:
👍 for adding the `doc` to `tixel_x` - but would you mind doing it for the others as well? The suggestion was a kind of hint